Model/Datasource/LdapSource.php

```diff
@@ -1469,32 +1481,15 @@ public function boolean() {
 	}
 
 /**
- * Returns the count of records
+ * Returns an calculation
  *
  * @param model $model
  * @param string $func Lowercase name of SQL function, i.e. 'count' or 'max'
  * @param array $params Function parameters (any values must be quoted manually)
- * @return int Entry count
+ * @return null
```

Comment:
Why not void directly?

Reply:
I thought some IDEs or static analyzers report warnings about returning null if `@return void` is written in the doc block, as it is not allowed in PHP7 with `strict_types=1`.

Reply:
You would also not return anything then :) Which you are faking here anyway, right?

Reply:
The return value of `calculate()` is used in [Model::_findCount()](https://github.com/cakephp/cakephp/blob/cd6fba136161993eb75913c55e1a3bf3da036e52/lib/Cake/Model/Model.php#L3171). So I thought null is better than void.

Reply:
Ah I see :+1: